ios/engine/KMEI/KeymanEngine/Classes/KeymanWebViewController.swift

```diff
@@ -247,6 +247,31 @@ extension KeymanWebViewController {
     log.debug("Keyboard stub: \(stubString)")
     webView!.evaluateJavaScript("setKeymanLanguage(\(stubString));", completionHandler: nil)
   }
+    
+    func setLexicalModel(_ lexicalModel: InstallableLexicalModel) {
```

Comment:
Minor note - formatting's a bit off.  Team convention:  2 spaces per indent, not 4.  Xcode automatically does 4 until told otherwise, unfortunately.

Reply:
Yes, I've been correcting this lately as I run across it, but there's a bunch that non-standard right now. I  better change my Xcode preferences...